t/clean_test_dirs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Copyright (C) 2018-2019 GitHub, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses/ .
+
+test_description='test directory cleanup utility'
+
+if test $# -eq 1 && test ":$1" = ":--all"
+then
+	set -- --clean-all
```

Comment:
TIL `set --`!

Reply:
Glad to be of some small service!  This is a only WIP PR for now because I'm going to experiment with CI and Docker in some followup commits and may very well break stuff as I stumble through it.

Reply:
:D Excellent! Have fun!